app/views/calculation/report/finalSummaryReport.scala.html

```diff
@@ -45,43 +48,7 @@ <h1 class="heading-xlarge">
         </h1>
     </div>
 
-    @{if (!taxYear.isValidYear) {
-    <div class="notice-wrapper">
-        <div class="notice">
-            <strong class="bold-small">
-                {Messages("calc.base.summary.dateWarning", taxYear.calculationTaxYear)}
-            </strong>
-        </div>
-    </div>
-    }
-    }
-
-    <!-- Calculation Details Section -->
-    <section id="calcDetails">
-        <!-- Header -->
-        @summarySectionHeaderHelper(Messages("calc.resident.summary.calcDetailsHeadingDate", taxYear.taxYearSupplied))
-
-        <!-- Data Rows -->
-        @summaryNumericRowHelper("gain",gainMessage(result.gain),result.gain)
-        @summaryNumericRowHelper("deductions", Messages("calc.resident.summary.deductions"), result.deductions, None,
-        Some(Seq(
-        (Messages("calc.resident.summary.deductionsDetails.aeaUsed"), MoneyPounds(result.aeaUsed, 0).quantity),
-        (Messages("calc.resident.summary.deductionsDetails.broughtForwardLossesUsed", taxYear.taxYearSupplied), MoneyPounds(result.broughtForwardLossesUsed, 0).quantity)
-        )))
-
-        @summaryNumericRowHelper("chargeableGain", if(result.chargeableGain < 0) {
-        Messages("calc.resident.summary.chargeableLoss")} else {Messages("calc.resident.summary.chargeableGain")},
-        result.chargeableGain)
-
-        @if(!result.secondRate.isDefined && !result.secondBand.isDefined) {
-        @summaryGainAndRateHelper("gainAndRate", Messages("calc.resident.summary.taxRate"), result.firstBand,
-        result.firstRate, None, None)
-        } else {
-        @summaryGainAndRateHelper("gainAndRate", Messages("calc.resident.summary.taxRate"), result.firstBand,
-        result.firstRate, result.secondBand, result.secondRate)
-        }
-        @summaryNumericRowHelper("aeaRemaining", Messages("calc.resident.summary.aeaRemaining", taxYear.calculationTaxYear), 0)
-    </section>
+    @finalSummaryPartial(gainAnswers, deductionAnswers, result, taxYear, 100, 100)
```

Comment:
I think you'll need to replace this (....., 100, 100) with the costs and deductions parameters :)